README.rst

```diff
@@ -2095,7 +2095,7 @@ The example below will add two rules to the default list of PSR2 set rules:
         ->in(__DIR__)
     ;
 
-    return PhpCsFixer\Config::create()
+    return (new PhpCsFixer\Config('foo'))
```

Comment:
I would not use a name in the documentation as it tends to imply that it is needed while it is optional.

Reply:
I hesitated: maybe it's too much information at a time for first timers. I'll remove that.

Reply:
I think you reverted the removal :)